Overview.html

```diff
@@ -1371,8 +1372,20 @@ <h2 id="fetching"><span class="secno">5 </span>Fetching</h2>
 <p>To perform a <dfn id="concept-fetch" title="concept-fetch">fetch</dfn> using <var title="">request</var>, run
 the steps below. An ongoing <a href="#concept-fetch" title="concept-fetch">fetch</a> can be
 <dfn id="concept-fetch-terminate" title="concept-fetch-terminate">terminated</dfn> with reason <var title="">reason</var>,
-which must be one of <i title="">end-user abort</i>, <i title="">fatal</i>, or
-<i title="">timeout</i>.
+which must be one of <i title="">end-user abort</i>, <i title="">fatal</i>, <i title="">timeout</i>, or
+<i title="">garbage collection</i>.
+
+<p>The user agent may be asked to <dfn id="concept-fetch-suspend" title="concept-fetch-suspend">suspend</dfn> the ongoing fetch.
+The user agent may either accept or ignore the suspension request. The susppended fetch can be
+<dfn id="concept-fetch-resume" title="concept-fetch-resume">resumed</dfn>. The user agent should ignore the suspension request
+if the ongoing fetch is updating the response in the HTTP cache for the request.
+
+<p class="note no-backref">The user agant does not update the entry in the HTTP cache for a
+<a href="#concept-request" title="concept-request">request</a> if request's cache mode is "no-store".
+
+<p class="note no-backref">The user agent does not update the entry in the HTTP cache for a
+<a href="#concept-request" title="concept-request">request</a> if `<code>Cache-Control: no-store</code>` HTTP
+header appears in the response. <a href="#refsHTTP">[HTTP]</a>
```

Comment:
Can you combine these two notes into one? Seems a simple "or" would suffice to avoid a lot of redundant text.